src/Cutscenes.ts

```diff
@@ -1637,6 +1637,146 @@ export class Cutscenes<TEightBittr extends FullScreenPokemon> extends EightBittr
     }
 
     /**
+     * Cutscene for a Quick Attack attack in battle.
+     * 
+     * @param settings   Settings used for the cutscene.
+     * @param args   Settings for the routine.
+     */
+    public cutsceneBattleAttackQuickAttack(settings: IBattleCutsceneSettings, args: IBattleAttackRoutineSettings): void {
+        const attackerName: string = args.attackerName;
+        const defenderName: string = args.defenderName;
+        const attacker: IThing = this.EightBitter.BattleMover.getThing(attackerName) as IThing;
+        const defender: IThing = this.EightBitter.BattleMover.getThing(defenderName) as IThing;
+        const direction: number = attackerName === "player" ? 1 : -1;
+        const menu: IMenu = this.EightBitter.MenuGrapher.getMenu("BattleDisplayInitial") as IMenu;
+
+        let xvel: number = -7 * direction;
+        let dt: number = 20;
+        const movement: TimeHandlr.ITimeEvent = this.EightBitter.TimeHandler.addEventInterval(
+            (): void => {
+                this.EightBitter.physics.shiftHoriz(attacker, xvel);
+            },
+            1,
+            Infinity);
```

Comment:
Instead of using `Infinity` here and cancelling the event later, you can compute the actual number of times it should run and just pass that.